Xamarin.Forms.Core/Button.cs

```diff
@@ -8,16 +8,13 @@
 namespace Xamarin.Forms
 {
 	[RenderWith(typeof(_ButtonRenderer))]
-	public class Button : View, IFontElement, ITextElement, IBorderElement, IButtonController, IElementConfiguration<Button>, IPaddingElement, IBorderController, IImageController, IViewController
+	public class Button : View, IFontElement, ITextElement, IBorderElement, IButtonController, IElementConfiguration<Button>, IPaddingElement, IImageController, IViewController, IButtonElement, IBorderOrCornerRadius, IImageElement
 	{
 		const double DefaultSpacing = 10;
-		const int DefaultBorderRadius = 5;
-		const int DefaultCornerRadius = -1;
 
-		public static readonly BindableProperty CommandProperty = BindableProperty.Create(nameof(Command), typeof(ICommand), typeof(Button), null, propertyChanging: OnCommandChanging, propertyChanged: OnCommandChanged);
+		public static readonly BindableProperty CommandProperty = ButtonElement.CommandProperty;
```

Comment:
what about a `CommandElement` ? does it makes sense ? would it be reused elsewhere ?